README.md

```diff
@@ -65,7 +65,8 @@ why these tests are disabled by default.
    (run tests with `./docker-tests.sh`).
  * `monitor`, tests sentry reporting, statsum submission and logging, requires
    credentials to run successfully.
-
+ * `docker`, tests docker-engine, disabled because tests requires docker
+   installed and must run as root (run tests with `./docker-tests.sh`).
```

Comment:
Isn't the current user being in the `docker` user group enough?

Reply:
nope, we need root because we listen on port 80 of the gateway IPs that docker creates for the isolated networks, we need this to be able to support attaching proxies from plugins.